sormas-api/src/main/java/de/symeda/sormas/api/user/UserRight.java

```diff
@@ -470,7 +470,8 @@
 			CONTACT_SUPERVISOR,
 			CONTACT_OFFICER,
 			EVENT_OFFICER,
-			COMMUNITY_OFFICER
+			COMMUNITY_OFFICER,
+			LAB_USER
```

Comment:
Why did we have to add this right? Would be good to at least have a comment in the #5644 issue

Reply:
we're fixing within this PR also #4694